cpp/data/eigerfan/src/EigerFan.cpp

```diff
@@ -95,7 +101,9 @@ EigerFan::EigerFan()
 EigerFan::EigerFan(EigerFanConfig config_)
 : ctx_(config_.num_zmq_threads),
   controlSocket(ctx_, ZMQ_ROUTER),
-  forwardSocket(ctx_, ZMQ_PUSH) {
+  forwardSocket(ctx_, ZMQ_PUSH),
+  broker(BROKER_INPROC_ENDPOINT, 8)
```

Comment:
Should this be either a configuration, or else a #define so that you can use the same value for both constructors and there is not a magic number present in the code.